pkg/autoscaler/stats_reporter.go

```diff
@@ -167,6 +167,13 @@ type Reporter struct {
 	initialized bool
 }
 
+func valueOrEmpty(l string) string {
+	if l == "" {
+		return "empty"
```

Comment:
How about using `ValueUnknown` from `knative/pkg/metrics/metricskey/constants.go`: https://github.com/knative/pkg/blob/master/metrics/metricskey/constants.go#L48

Reply:
Done.  
(Push force with the update.)